lib/cloudfrontUtil.js

```diff
@@ -44,9 +82,9 @@ function getSignedUrl(cfUrl, params) {
   parsedUrl = url.parse(cfUrl, true);
   parsedUrl.search = '';
   _.extend(parsedUrl.query, {
-    'Expires': policy.expireTime,
-    'Policy': new Buffer(policy.toJSON()).toString('base64'),
-    'Signature': normalizeSignature(signature),
+    'Expires': params.expireTime / 1000,
+    'Policy': policy,
+    'Signature': sig,
```

Comment:
I like the addition of `genPolicy` but why the changes to `Expires` and `Signature`? It doesn't look like there needs to be this many modifications to `getSignedUrl`. Can you minimize the changes to this function? I think all that's needed is to update

`'Policy': new Buffer(policy.toJSON()).toString('base64')` ==> `'Policy': genPolicy(cfUrl, params)`